src/app/public/modules/wait/wait.service.ts

```diff
@@ -9,13 +9,22 @@ import {
 } from '@skyux/core';
 
 import {
-  SkyWaitPageComponent
-} from './wait-page.component';
+  defer,
+  Observable
+} from 'rxjs';
+
+import {
+  finalize
+} from 'rxjs/operators';
```

Comment:
We'll need to keep this library backwards compatible with Builder v.1 for a few more months, which means we'll have to use the "old" import structure from RxJS v.5.
```
import {
  Observable
} from 'rxjs/Observable';

import 'rxjs/add/operators/finalize';
```